shell.nix

```diff
@@ -0,0 +1,10 @@
+{ nixpkgs ? import <nixpkgs> { } }:
+
+with nixpkgs;
+
+let  mkEnv = { stdenv, python27, pythonPackages, openssl }:
+  stdenv.mkDerivation {
+    name = "mantl-env";
+    buildInputs = [ python27 pythonPackages.pyyaml openssl terraform packer ansible2 openssh ];
```

Comment:
Correct, I will add vagrant and netaddr, and doc build stuff. Not sure if git is needed to operate cluster/build docs.